src/internal/modules/charge-information/lib/charge-categories/constants.js

```diff
@@ -0,0 +1,111 @@
+'use-strict';
+
+const CHARGE_CATEGORY_STEPS = {
+  description: 'description',
+  source: 'source',
+  loss: 'loss',
+  volume: 'volume',
+  waterAvailability: 'water-availability',
+  waterModel: 'water-model',
+  isAdditionalCharges: 'additional-charges',
+  isAdjustments: 'adjustments'
+};
+
+const getStepKeyByValue = value => Object.keys(CHARGE_CATEGORY_STEPS).find(key => CHARGE_CATEGORY_STEPS[key] === value);
+
+const CHARGE_CATEGORY_FIRST_STEP = CHARGE_CATEGORY_STEPS.description;
+
+const SOURCES = {
+  tidal: 'tidal',
+  nonTidal: 'non-tidal'
+};
+
+const WATER_MODEL = {
+  noModel: 'no model',
+  tier1: 'tier 1',
+  tier2: 'tier 2'
+};
+
+const LOSS_CATEGORIES = {
+  high: 'high',
+  medium: 'medium',
+  low: 'low'
+};
+
+const WATER_AVAILABILITY = {
+  available: 'available',
+  restricted: 'restricted availablity or no availability'
+};
+
+const YES_NO = {
+  // true: true,
```

Comment:
What is this doing? the //true: true doesn't look right. Could you kindly re-check what's happening here?

Reply:
changed this to the shape below to act as a double purpose mapper and feed the options on the form. 
`const YES_NO = [
  {
    label: 'yes',
    value: true
  },
  {
    label: 'no',
    value: false
  }
];`